hydra-node/json-schemas/api.yaml

```
@@ -571,6 +580,34 @@ components:
         timestamp:
           $ref: "api.yaml#/components/schemas/UTCTime"
 
+    PeerHandshakeFailure:
+      type: object
+      required:
+        - tag
+        - remoteHost
+        - ourVersion
+        - theirVersions
+        - seq
+        - timestamp
+      properties:
+        tag:
+          type: string
+          enum: ["PeerHandshakeFailure"]
+        remoteHost:
+          type: object
+          $ref: "api.yaml#/components/schemas/IP"
+        ourVersion:
+          type: integer
+          minimum: 0
+        theirVersions:
+          type: array
+          items:
+            type: integer
```

```suggestion
            type: integer
            minimum: 0
```

Or use a named schema for this "HydraNetworkProtocolVersion"`?